jsk_rosbag_tools/requirements.in

```diff
@@ -0,0 +1,4 @@
+moviepy
```

Comment:
could specify version? to prevent install latest version which is not compatible with this source tree.

Reply:
I no longer use catkin virtualenv. So, I add it to package.xml.
https://github.com/jsk-ros-pkg/jsk_common/blob/9b7b79ed30773e275fa8ebc0aab788248bfb80dd/jsk_rosbag_tools/package.xml